packages/tools/src/stateManagement/segmentation/polySeg/Surface/surfaceComputationStrategies.ts

```diff
@@ -0,0 +1,157 @@
+import { Types } from '@cornerstonejs/core';
+import {
```

Comment:
Can you import with type - just that it avoids circular dependencies, and I see you export a type down below, so I think it should have the types.